docs/api-docs/cart-and-checkout/custom-order-confirmation.md

```diff
@@ -0,0 +1,92 @@
+
+1. Navigate and locate the Open Source Checkout folder using the terminal, and run `npm ci` to download dependencies
+2. Run `npm run build` to generate a `/dist/` folder with all the files
+
+When using Open Source Checkout to view changes and test implementations through the UI, deploy, and install the custom confirmation page. More details are below.
+
+### Properties available on the custom order confirmation page
+When a store uses a custom confirmation page, some properties are attached to the `window.checkoutConfig` object on the checkout page. These properties are:
+
+* `containerId`: ID of the HTML component where you load the checkout app.
+* `orderId`: if present, it means we should render an order confirmation page for the given order Id
+* `checkoutId`: If present, the ID of the active checkout. You can't have both `orderId` and `checkoutId`
+
+To use the data exposed in the window object, include relevant code in your loader file. Here is an example of this code :
+
+```js
+ReactDOM.render(
+  <Checkout checkoutId={ window.checkoutConfig.checkoutId } />,
+  document.getElementById(window.checkoutConfig.containerId)
+);
+```
+
+## Hosting a custom order confirmation page
+
+You will need to host the custom confirmation file online so it can be served by the store. You can use a hosting service, such as [Amazon S3](https://aws.amazon.com/s3/). Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
+
+### Using WebDAV to host a custom order confirmation page
+
+You can upload a custom checkout to your store's server using WebDAV. The instructions to upload the `/dist` folder to the BigCommerce server using WebDav are below:
+
+1. Before proceeding, ensure you have downloaded [Cyberduck](https://cyberduck.io/), our recommended WebDAV client.
+**Note:**  For more information on how to use Cyberduck, refer to the [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV) page.
+2. From your store control panel, navigate to **Server Settings** > **File Access (WebDAV)**.
+    - To automatically connect with Cyberduck, see [Single-click Login](https://support.bigcommerce.com/s/article/File-Access-WebDAV#login).
+    - To manually connect with Cyberduck, see [Connecting with Cyberduck Manually](https://support.bigcommerce.com/s/article/File-Access-WebDAV#manual).
+3. From Cyberduck, enter the `/content` folder and create a new folder named *checkout*.
+4. Navigate into the `/dist` folder in your checkout project.
+5. Copy the contents of the `/dist` folder and paste it into the checkout folder.
+
+## Installing a custom order confirmation page  
+
+To install a custom order confirmation page on a store, follow these steps:
+
+1. Navigate to **Advanced Settings > Checkout** in your store's control panel.
+2. Under **Checkout Type**, select **Custom Checkouts**.
```

Comment:
this is not correct, they are free to choose any Checkout Type.
Once the experiment is rolled out, there should be a panel that reads "Order Confirmation Settings", and there they can enter a Script URL under the Custom Order Confirmation Page settings.